packages/native/src/polywrap_native.rs

```diff
@@ -0,0 +1,27 @@
+use std::sync::Arc;
+
+use polywrap_client::core::{file_reader::SimpleFileReader, package::WrapPackage, wrapper::Wrapper};
+use polywrap_wasm::{wasm_package::WasmPackage, wasm_wrapper::WasmWrapper};
+
+use crate::{
+    error::FFIError,
+    package::FFIWrapPackage, wrapper::FFIWrapper,
+};
+
+pub fn ffi_wrap_package_from_bytecode(bytes: &[u8]) -> Result<Arc<FFIWrapPackage>, FFIError> {
+    let wasm_package =
+        WasmPackage::from_bytecode(bytes.to_vec(), Arc::new(SimpleFileReader::new()), None);
```

Comment:
if we have the bytes of the module already fo we really need the file reader? i guess we can modify the function `from_bytecode` to only accept an array of `u8`?